src/handlerResponses.ts

```diff
@@ -0,0 +1,9 @@
+interface ClusterStatusResponse {
+    instanceId: string;
+    clusterStatus: string;
+}
+
+interface AddElasticsearchNodeResponse {
+    instanceId: string;
+    expectedClusterSize: number;
```

Comment:
🤓 this might be excessive, however it seems like typescript can set properties to `readonly` which would potentially guard against the mutability of objects and nested objects...

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze

The idea that this is possible excites me
```
interface ClusterStatusResponse {
    readonly instanceId: string;
    readonly clusterStatus: string;
}
```

Reply:
Using const is probably sufficient for our purposes here, but I agree that it's neat that you can do this!